libsupport/src/Plugin.cpp

```diff
@@ -0,0 +1,202 @@
+#include "katana/Plugin.h"
+
+#include <dlfcn.h>
+
+#include <cstdlib>
+#include <unordered_map>
+
+#include <boost/filesystem.hpp>
+#include <katana/Logging.h>
+#include <katana/Result.h>
+
+namespace {
+
+std::vector<std::string>
```

Comment:
We have `katana::SplitView` in `Strings.h` and, in general, `Strings.h` is the place for basic string utilities when you don't really care about large inputs, etc but just want convenience, which I think applies here.

Reply:
Oops. 👍🏻